src/components/browser/BrowseHierarchy.tsx

```diff
@@ -90,9 +90,10 @@ export class BrowseHierarchy extends Component<Props, State> {
           <Input css="" value={search} onChange={this.onSearchChange} placeholder="search..." />
           <br />
 
-          {asset.hierarchy.map(h => {
+          {asset.hierarchy.map((h) => {
             return (
               <AssetHierarchyList
+                key={h.value}
```

Comment:
this is the only real frontend change -- the lint rules complained that key was missing

All other frontend changes cosmetic (and automatic)